src/components/seo.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * SEO component that queries for data with
+ *  Gatsby's useStaticQuery React hook
+ *
+ * See: https://www.gatsbyjs.org/docs/use-static-query/
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import Helmet from "react-helmet";
+import { useStaticQuery, graphql } from "gatsby";
+
+function SEO({ lang, meta }) {
+  const { site } = useStaticQuery(
+    graphql`
+      query {
+        site {
+          siteMetadata {
+            title
+            author
+            description
+          }
+        }
+      }
+    `
+  );
+
+  return (
+    <Helmet
+      htmlAttributes={{
+        lang
+      }}
+      title={site.siteMetadata.title}
+      meta={[
+        {
+          name: `description`,
+          content: site.siteMetadata.description
+        },
+        {
+          property: `og:title`,
+          content: site.siteMetadata.title
+        },
+        {
+          property: `og:description`,
+          content: site.siteMetadata.description
+        },
+        {
+          property: `og:type`,
+          content: `website`
+        },
+        {
+          name: `twitter:card`,
+          content: `summary`
+        },
+        {
+          name: `twitter:creator`,
+          content: site.siteMetadata.author
+        },
+        {
+          name: `twitter:title`,
+          content: site.siteMetadata.title
+        },
+        {
+          name: `twitter:description`,
+          content: site.siteMetadata.description
+        }
+      ].concat(meta)}
+    />
+  );
+}
+
+SEO.defaultProps = {
```

Comment:
These defaults are overwritten by the values returned by the graphql query, right?

Reply:
Not really, it's just the default language and empty meta here. GraphQL provides title, description and author.